etc/amazon-corretto-crypto-provider-jdk15.security

```diff
@@ -0,0 +1,36 @@
+#
+# List of providers and their preference orders
+
+# Note that we can't just insert ours at the top - we need to restate all
+# providers that are being shifted down to a different index (which is to say,
+# all of them).
+#
+# To ensure this file works on all possible platforms we do include platform specific providers.
+
+security.provider.1=com.amazon.corretto.crypto.provider.AmazonCorrettoCryptoProvider
+security.provider.2=SUN
+security.provider.3=SunRsaSign
+security.provider.4=SunEC
+security.provider.5=SunJSSE
+security.provider.6=SunJCE
+security.provider.7=SunJGSS
+security.provider.8=SunSASL
+security.provider.9=XMLDSig
+security.provider.10=SunPCSC
+security.provider.11=JdkLDAP
+security.provider.12=JdkSASL
+security.provider.13=SunMSCAPI
+security.provider.14=Apple
+security.provider.15=SunPKCS11
```

Comment:
out of curiosity, what's this list based on? how are the priorities decided?

Reply:
This is based on the current provider list in Corretto-17